### PR TITLE
Pattern_code

### PR DESCRIPTION
**Pull request by codervipul775**

### CODE REVIEW

**Review by hasan-py — CHANGES_REQUESTED** [review]
Please give a convenient PR title.

**Comment by codervipul775** [comment]
updated.

File: README.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ Copy the next template into your file, delete the boilerplate data and fill the
 
 ```
 ---
-name: YOURNAME
-institution/company: INSTITUTION_NAME
-github: USERNAME
+name: Vipul
+institution/company: Newton School of Technology
```
Comment (hasan-py) [inline README.md:138]: Why you updated the main readme file?

**Review by hasan-py — CHANGES_REQUESTED** [review]

**Review by hasan-py — APPROVED** [review]
LGTM, Thanks!

**Comment by codervipul775** [comment]
Why this repository is excluded from hactoberfest

**Comment by DhanushNehru** [comment]
Hi @codervipul775  this is just a template repo to fork and raise a PR

Checkout repos [here to contribute that counts towards hacktoberfest](https://medium.com/@dhanushnehru/kickstart-hacktoberfest-with-these-exciting-open-source-projects-to-contribute-to-03a61cc43d0d?sk=28f95a04e212f16c5f7c3ba4b3fd5f31)  and join discord for more discussions

**Pull request merged**